### PR TITLE
experimental: Paste from webflow 5

### PR DESCRIPTION
**Pull request by kof**
## Description

When parsing webflow presets:
- skip selectors with combinators
- support states

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Comment by TrySound** [comment]
> msTextSizeAdjust

we don't support ms prefixes already, should be excluded

> webkitTextSizeAdjust

this is unprefixed for a long time

**Comment by kof** [comment]
yeah, we will clean up that stuff in the next prs

File: packages/css-data/src/parse-css.test.ts [hunk 1]
```diff
@@ -124,6 +158,44 @@ describe("Parse CSS", () => {
...
           ],
         }
-      `);
+    `);
+  });
+
+  test("parse child combinator", () => {
+    expect(parseCss(`a > b { color: #ff0000 }`)).toMatchInlineSnapshot(`
+      {
+        "a>b": [
```
Comment (TrySound) [inline packages/css-data/src/parse-css.test.ts:167]: what is the plan with these?

Comment (kof) [reply packages/css-data/src/parse-css.test.ts:199]: alright lets skip those for now during generation

Comment (kof) [reply packages/css-data/src/parse-css.test.ts]: done

**Review by TrySound — APPROVED** [review]

**Pull request merged**